ui/src/components/scroll-area/QScrollArea.js

```diff
@@ -154,97 +244,162 @@ export default Vue.extend({
       return this.$refs.target
     },
 
+    getScroll () {
+      const info = {}
+
+      axisList.forEach(axis => {
+        const data = this.scrollComputed[ axis ]
+
+        info[ axis + 'Position' ] = data.position
+        info[ axis + 'Percentage' ] = data.percentage
+        info[ axis + 'Size' ] = data.size
+        info[ axis + 'ContainerSize' ] = this.container[ axis ]
+      })
+
+      return info
+    },
+
     getScrollPosition () {
-      return this.scrollPosition
+      return {
```

Comment:
Should return the same as in previous version. For non-breaking changes purposes, let's add a getVerticalScrollPosition() method as well, unless you have a better idea.